build.sbt

```
@@ -882,6 +899,20 @@ lazy val tests =
             Seq((Test / sourceDirectory).value / "scala-2.12+")
           case _ => Nil
         }
+      },
+      // temp hack
+      Test / unmanagedSourceDirectories ++= {
+        // unit-tests/native/src/test
+        val testDir = (Test / sourceDirectory).value
+        val rootTestDir =
+          testDir.getParentFile.getParentFile.getParentFile
```

We can also simplify this one
```suggestion
        // unit-tests/native
        val testDir = (Test / baseDirectory).value
        val rootTestDir = testDir.getParentFile
```

---

Used the baseDirectory as suggested.